cumulusci/salesforce_api/metadata.py

```diff
@@ -364,7 +364,7 @@ def __init__(self, task, package_zip, purge_on_delete=None, api_version=None):
         self.package_zip = package_zip
 
     def _set_purge_on_delete(self, purge_on_delete):
-        if purge_on_delete == False or purge_on_delete == "false":
+        if purge_on_delete is False or purge_on_delete == "false":
```

Comment:
There's a subtle change in behavior here:

```
>>> 0 == False
True
>>> 0 is False    
False
```

I doubt it matters here, but `if not purge_on_delete or purge_on_delete == "false"` might be preferable.